### PR TITLE
[flutter_tts] Fix bug and support setting speech volume

### LINKED ISSUE #257: [flutter_tts] PlatformException when `FlutterTts.setLanguage()` is called
I've tested with flutter_tts/example app.
When `setLanaguage()` is called through dropdown menu, below error happens.

```
E/ConsoleMessage( 4431): [ERROR:flutter/lib/ui/ui_dart_state.cc(209)] Unhandled Exception: PlatformException(-1, Not supported method, null, null)
#0      StandardMethodCodec.decodeEnvelope (package:flutter/src/services/message_codecs.dart:607:7)
#1      MethodChannel._invokeMethod (package:flutter/src/services/platform_channel.dart:156:18)
<asynchronous suspension>

```

flutter-tizen doctor -v
```
[✓] Flutter (Channel unknown, 2.5.1, on Ubuntu 20.04.2 LTS 5.11.0-37-generic, locale ko_KR.UTF-8)
    • Flutter version 2.5.1 at /home/flutter_tizen/flutter-tizen/flutter
    • Upstream repository unknown
    • Framework revision ffb2ecea52 (6주 전), 2021-09-17 15:26:33 -0400
    • Engine revision b3af521a05
    • Dart version 2.14.2

[✓] Tizen toolchain - develop for Tizen devices
    • Framework revision d5bf5a5635 (3일 전), 2021-10-25 10:32:10 +0900
    • Engine revision 7e8adde964
    • Tizen Studio 4.1.1 at /home/tizen-studio
    • .NET SDK 5.0.400 at /usr/bin/dotnet

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0)
    • Android SDK at /home/Android/Sdk
    • Platform android-31, build-tools 31.0.0
    • Java binary at: /opt/android-studio-4.1/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)
    • All Android licenses accepted.

[✓] Android Studio (version 4.1)
    • Android Studio at /opt/android-studio-4.1/android-studio
    • Flutter plugin version 57.0.1
    • Dart plugin version 201.9380
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] Android Studio
    • Android Studio at /opt/android-studio
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 1.8.0_242-release-1644-b3-6222593)

[✓] VS Code (version 1.57.1)
    • VS Code at /usr/share/code
    • Flutter extension version 3.24.0

[✓] Connected device (3 available)
    • Tizen m-0727-2 (mobile)        • emulator-26101  • flutter-tester • Tizen 6.0 (emulator)
```

### PR DESCRIPTION
**Pull request by HakkyuKim**
closes #257.

TODO:
- [x] Support `set speech volume`. 
    `tts.h` doesn't have apis that can control the tts volume, I could increase the volume with sound manager but I'm looking for something that wouldn't affect other systems. -> tts internally acquire and relinquish focus on voice audio stream.

### CODE REVIEW

**Review by bbrto21 — APPROVED** [review]

File: packages/flutter_tts/example/tizen/Runner.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Tizen.NET.Sdk/1.1.5">
+﻿<Project Sdk="Tizen.NET.Sdk/1.1.6">
```
Comment (bwikbs) [inline packages/flutter_tts/example/tizen/Runner.csproj:1]: Just a question, is this change necessary?

**Review by bwikbs — APPROVED** [review]
:+1:

File: packages/flutter_tts/example/tizen/Runner.csproj (hunk 1 above)
Comment (HakkyuKim) [reply packages/flutter_tts/example/tizen/Runner.csproj:1]: There weren't any problems that I know of, I just updated it to a latest compatible version.
